sample_dataset/metabase/sample_dataset/generate.clj

```diff
@@ -194,20 +204,33 @@
   (doto (Date.)
     (.setTime (apply min (map #(.getTime ^Date %) dates)))))
 
+(defn- sometimes
+  [p f]
+  (when (> p (rand))
+    (f)))
```

Comment:
So... this function sometimes does `f`, and `p` is the probability of doing it? That's not clear at all until you read the actual code for the function. I would vote in favor of a slightly more verbose name that more adequately summarizes what is going on. `do-sometimes-with-probability` or something like that is a lot clearer. At any rate a docstring would have *also* been a good way to clear this up

Reply:
+1